modules/bash-commons/src/array.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
+
+set -e
+
+# Returns 0 if the given item (needle) is in the given array (haystack); returns 1 otherwise.
+function array_contains {
+  local readonly needle="$1"
+  shift
+  local readonly haystack=($@)
```

Comment:
probably want to "$@" to avoid inadvertently splitting

Reply:
Ah, good catch. That's definitely a bug. Added a fix, including test cases to catch it in the future: https://github.com/gruntwork-io/bash-commons/pull/1/commits/a666074627dc2225270588275b17e748f744ee1a